src/components/treeInfo/index.tsx

```diff
@@ -143,39 +144,43 @@ const TreeInfo: React.FC<TreeProps> = ({
 
       {loggedIn ? (
         <>
-          {userOwnsTree ? (
-            <UnadoptButton
-              danger
-              size={mobile ? 'middle' : 'large'}
-              onClick={onClickUnadopt}
-            >
-              {t('actions.unadopt')}
-            </UnadoptButton>
-          ) : (
-            <Button
-              type="primary"
-              size={mobile ? 'middle' : 'large'}
-              onClick={onClickAdopt}
-              disabled={isAdopted}
-            >
-              {isAdopted ? t('actions.adopted') : t('actions.adopt')}
-            </Button>
-          )}
-
-          {userIsAdmin && (
```

Comment:
As a follow-up to this follow-up, I think when a new site entry is added to a site or an existing latest site entry is modified 
where the entry's `treePresent` value is false and the site is already adopted, we should automatically unadopt the site